cmd/init.go

```diff
@@ -186,21 +170,7 @@ func (o *initOptions) scaffolder(c *config.Config) (scaffold.Scaffolder, error)
 func (o *initOptions) postScaffold(c *config.Config) error {
 	switch {
 	case c.IsV1():
```

Comment:
In some places we keep checks for v1 projects, and in others we remove them. It would be better to centralize the check for unsupported project versions somewhere like `main.go` so we don't have distributed (error-prone) checks.

Reply:
You are right. We have the check the main go already.